content/en/docs/tasks/debug/debug-cluster/resource-usage-monitoring.md

```diff
@@ -59,8 +59,19 @@ Autoscaler. The monitoring pipeline fetches metrics from the kubelet and
 then exposes them to Kubernetes via an adapter by implementing either the
 `custom.metrics.k8s.io` or `external.metrics.k8s.io` API. 
 
-[Prometheus](https://prometheus.io), a CNCF project, can natively monitor Kubernetes, nodes, and Prometheus itself.
-Full metrics pipeline projects that are not part of the CNCF are outside the scope of Kubernetes documentation.  
+Kubernetes is designed to work with [OpenMetrics](https://openmetrics.io/), 
+which is one of the
+[CNCF Observability and Analysis - Monitoring Projects](https://landscape.cncf.io/card-mode?category=monitoring&project=graduated,incubating,member,no&grouping=category&sort=stars),
+built upon and carefully extending [Prometheus exposition format](https://prometheus.io/docs/instrumenting/exposition_formats/)
+in almost 100% backwards-compatible ways.
+
+If you glance over at 
+[CNCF Landscape](https://landscape.cncf.io/card-mode?category=monitoring&project=graduated,incubating,member,no&grouping=category&sort=stars), 
+you can observe that the Kubernetes project doesn't provide a recommendation 
+of any projects to work on, as it is totally up to the users to find the 
+project that suits them the best. If someone is to make a full metrics pipeline,
+they need to design and deploy their own infrastructure platform, of which
+Kubernetes is only one part.
```

Comment:
I'd write:
```suggestion
you can see a number of monitoring projects that can work with Kubernetes by _scraping_
metric data and using that to help you observe your cluster. It is up to you to select the tool
or tools that suit your needs. The CNCF landscape for observability and analytics includes a
mix of open-source software, paid-for software-as-a-service, and other commercial products.

When you design and implement a full metrics pipeline you can make that monitoring data
available back to Kubernetes. For example, a HorizontalPodAutoscaler can use the processed
metrics to work out how many Pods to run for a component of your workload.
```

Reply:
Ok, this sounds way better than mine 😂. @NitishKumar06 what's your opinion on this??

Reply:
I would go with that suggestion @ShivangShandilya (I'm writing as a SIG Docs reviewer).

Reply:
Proceed with the suggestions given by @sftim